lib/controllers/issueish-list-controller.js

```diff
@@ -51,12 +49,13 @@ export class BareIssueishListController extends React.Component {
 
     resultFilter: PropTypes.func,
     onOpenIssueish: PropTypes.func.isRequired,
+    onOpenReviews: PropTypes.func.isRequired,
     onOpenMore: PropTypes.func,
 
     emptyComponent: PropTypes.func,
     workspace: PropTypes.object,
     endpoint: EndpointPropType,
-    workingDirectory: PropTypes.string,
+    workdir: PropTypes.string.isRequired,
```

Comment:
I'm getting a PropTypes error in the console:

```
/Users/annthurium/github/github/node_modules/prop-types/checkPropTypes.js:20 Warning: Failed prop type: The prop `workdir` is marked as required in `BareIssueishListController`, but its value is `undefined`.
    in BareIssueishListController (created by ReactRelayQueryRenderer)
    in ReactRelayQueryRenderer (created by IssueishSearchContainer)
    in IssueishSearchContainer (created by IssueishSearchesController)
    in div (created by IssueishSearchesController)
    in IssueishSearchesController (created by RemoteController)
    in RemoteController (created by ReactRelayQueryRenderer)
    in ReactRelayQueryRenderer (created by ObserveModel)
    in ObserveModel (created by RemoteContainer)
    in RemoteContainer (created by GitHubTabView)
    in div (created by GitHubTabView)
    in div (created by GitHubTabView)
    in GitHubTabView (created by GitHubTabController)
    in GitHubTabController (created by ObserveModel)
    in ObserveModel (created by GitHubTabContainer)
    in GitHubTabContainer (created by GitHubTabItem)
    in GitHubTabItem (created by PaneItem)
    in PaneItem (created by RootController)
    in RootController
```

Is the workdir required by all places in the component tree the instantiate the `IssueishListController`?  If so we need to make sure they're all passing that prop.  Else mark it as not required.

Reply:
Ooh good catch, forgot to clean this up when I lifted the logic to open `ReviewItem`s to the `IssueishSearchesController`... :fire:'d the unused prop